explainer.md

```diff
@@ -147,7 +150,18 @@ The root elements of each page are automatically associated.
 
 Note that the order in which the transition elements are painted can be configured by UA and/or developer stylesheets using z-index.
 
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/23): How should the default UA animation order these elements? And also handle a change in associated elements between the 2 pages. 
+- [Open question](https://github.com/WICG/shared-element-transitions/issues/23): How should the default UA animation order these elements? And also handle a change in associated elements between the 2 pages.
+
+### How are transition elements painted?
+During the transition a new stacking context (called uber-root) is created with the following hierarchy :
+
+```
+uber-root stacking context
+└─ root stacking context
```

Comment:
Perhaps root stacking context can be "page root stacking context" or something that makes it clear that it's a root for all things that the page itself may paint. Although technically the transition stacking context's elements are also styled by the page, so nevermind...

Reply:
It's actually better to call it "root stacking context" to make it obvious that we're talking about the existing concept of a page's root stacking context?

Reply:
It isn't clear to me what this is vs "root image"

Reply:
(we chatted and I think I understand it now. I'll add to this PR)